xds/internal/xdsclient/transport/transport_test.go

```diff
@@ -18,16 +18,62 @@
 package transport_test
 
 import (
+	"context"
+	"encoding/json"
+	"net"
 	"testing"
 
 	"google.golang.org/grpc"
-	"google.golang.org/grpc/internal/xds/bootstrap"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/insecure"
+	internalbootstrap "google.golang.org/grpc/internal/xds/bootstrap"
+	"google.golang.org/grpc/xds/bootstrap"
 	"google.golang.org/grpc/xds/internal/xdsclient/transport"
 	"google.golang.org/grpc/xds/internal/xdsclient/transport/internal"
 
 	v3corepb "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
 )
 
+const testDialerCredsBuilderName = "test_dialer_creds"
```

Comment:
Can you please move all of this code down so it immediately precedes the test that uses it?

It's unfortunate that the test body can't declare these types/etc, so let's at least put the code as close together as possible.

Reply:
Done.
Moved all this newly added section of code down such that it immediately preceded the test using it.
This follows readability best practices of declaring variables and functions immediately before usage.